### PR TITLE
JSCB metadata update

### PR DESCRIPTION
**Pull request by mnaguit**
update metadata for LC 21, 22 and 23

### CODE REVIEW

**Comment by ozym** [comment]
@samgeonet they are mini-recorders (they output digital counts, the Cusp-M box itself only gathers the data together into files, and adds a timing mechanism). The Cusp-M has no impact on the response (hence no need to be in delta -- other than perhaps to keep track of firmware versions for timing).

**Comment by ozym** [comment]
However, the clearly aren't CUSP3D recorders,  see the top of the recorders.csv file for an equivalent ...

```
Make,Sensor,Datalogger,Serial,Station,Location,Azimuth,Dip,Depth,Start Date,End Date
CSI,CUSP-Me,CUSP-Me,3DE-20733,TRAB,21,278,0,-1.2,2013-08-02T00:00:00Z,9999-01-01T00:00:00Z
CSI,CUSP-Me,CUSP-Me,3DE-20734,TRAB,22,98,0,-1.25,2013-08-02T00:00:00Z,9999-01-01T00:00:00Z
CSI,CUSP-Me,CUSP-Me,3DE-20735,TRAB,23,98,0,-5.9,2013-08-02T00:00:00Z,9999-01-01T00:00:00Z
```

**Comment by ozym** [comment]
@mnaguit you will need to remove the trailing zero from the site files line:

`JSCB,22,-43.518399,172.731880, ....`

should be

`JSCB,22,-43.518399,172.73188, ...`

The reason is that a check is made that the output matches the input, but as these are floating point numbers with no length, then the 0 will be stripped.  This is a known "feature" ...

**Comment by mnaguit** [comment]
Many thanks @ozym 👍 👍

File: install/recorders.csv [hunk 1]
```diff
@@ -422,6 +422,9 @@ CSI,CUSP3D,CUSP3D,42168,NSPS,21,251,0,0,2014-08-27T00:00:10Z,9999-01-01T00:00:00
 CSI,CUSP3D,CUSP3D,42169,IFPS,20,158,0,0,2014-08-12T00:05:00Z,9999-01-01T00:00:00Z
 CSI,CUSP3D,CUSP3D,42170,KOKS,20,118,0,0,2014-08-12T05:05:00Z,9999-01-01T00:00:00Z
 CSI,CUSP3D,CUSP3D,42172,NCHS,20,138,0,0,2014-08-27T00:00:10Z,9999-01-01T00:00:00Z
+CSI,CUSP3D,CUSP3D,42200,JSCB,21,252,0,-0.3,2014-06-15T00:00:00Z,9999-01-01T00:00:00Z
+CSI,CUSP3D,CUSP3D,42204,JSCB,22,252,0,-2.5,2014-06-15T00:00:00Z,9999-01-01T00:00:00Z
+CSI,CUSP3D,CUSP3D,42207,JSCB,23,252,0,-2.5,2014-06-15T00:00:00Z,9999-01-01T00:00:00Z
```
Comment (ozym) [inline install/recorders.csv]: From the docs, it looks like a CUSP-M4 is the model that is deployed.

I suggest calling it:

`CSI,CUSP-M4,3D-42200,JSCB,21,....`

File: assets/recorders.csv [hunk 2]
```diff
@@ -459,6 +459,9 @@ CSI,CUSP3D,42168,15195,
 CSI,CUSP3D,42169,15196,
 CSI,CUSP3D,42170,15197,
 CSI,CUSP3D,42172,15198,
+CSI,CUSP3D,42200,15621,
+CSI,CUSP3D,42204,15622,
+CSI,CUSP3D,42207,15623,
```
Comment (ozym) [inline assets/recorders.csv]: suggest,

`CSI,CUSP-M4,3D-42200,15621,`

etc

**Comment by mnaguit** [comment]
Hi @salichon @samgeonet @brianf3rris @jristau @ozym 
Can you please do another check for this PR? Thanks.

**Comment by mnaguit** [comment]
Hi @ozym . One final check on this. Changes have been made, as per your review. Can I merge this PR? Thanks.

**Comment by ozym** [comment]
In the streams file, the 200 Hz streams should be triggered over this time period.

**Comment by mnaguit** [comment]
Thanks @ozym . Modification on streams.csv has been made. For your final review.

**Comment by mnaguit** [comment]
Hi @salichon @jristau @samgeonet @ozym 
Changes have been made for this PR. Can you please do another round of checking? Thanks.

**Pull request merged**